nusight2/src/client/components/vision/image.ts

```diff
@@ -18,6 +18,11 @@ export interface ElementImage extends BaseImage {
   element: HTMLImageElement
 }
 
+export interface BitmapImage extends BaseImage {
+  type: 'bitmap'
+  bitmap: ImageBitmap
```

Comment:
Perhaps we should rename (in a separte PR?) `data` + `element` + `bitmap` to `image` so that we don't have to duplicate code e.g. needing to add basically a duplicate method `bitmapTexture` below.

Reply:
Done. Tried to merge all the `...Texture` methods but the data one needs to call `get()` on its image since the image is a computed. So I've merged the element and bitmap ones, keeping data separate.